src/plugins/workspace/public/utils.ts

```diff
@@ -32,14 +37,14 @@ export const getUseCaseFromFeatureConfig = (featureConfig: string) => {
 
 export const isFeatureIdInsideUseCase = (
   featureId: string,
-  featureConfig: string,
+  useCaseId: string,
   useCases: WorkspaceUseCase[]
 ) => {
-  const useCase = useCases.find(({ id }) => id === getUseCaseFromFeatureConfig(featureConfig));
-  if (useCase) {
-    return useCase.features.includes(featureId);
-  }
-  return false;
+  const availableFeatures =
+    useCaseId === ALL_USE_CASE_ID
+      ? useCases.reduce<string[]>((previous, { features }) => previous.concat(features ?? []), [])
+      : useCases.find(({ id }) => id === useCaseId)?.features ?? [];
+  return availableFeatures.includes(featureId);
```

Comment:
if an app have not register to any use case, will it be visible/accesable in all use case?

Reply:
Based current implementation, the all use case will match all features inside registered use cases. If an app not register to any use case, it still can be accessible if it match below conditions:
1. `workspaceAvailability` is `outsideWorkspace` 
2. `navLinkStatus` equal `hidden`
3. `chromeless` equal `true`